doc/examples/reconst_dki.py

```diff
@@ -228,19 +219,59 @@
    **Diffusion tensor measures obtain from the diffusion tensor estimated from
    DKI (upper panels) and DTI (lower panels).**.
 
-From the figure, we can see that the DT standard diffusion measures from DKI
-are noisier than the DTI measurements. This is a well known pitfall of DKI
-[NetoHe2014]_. Since it involves the estimation of a larger number of
-parameters, DKI is more sensitive to noise than DTI. Nevertheless, DT diffusion
-based measures were shown to have better precision (i.e. less sensitive to
-bias) [Veraa2011]_.
+From the figure, we can see that the standard diffusion measures from DKI has
+similar constrasts than the standard diffusion measures from DTI.
+
+In addition to the standard diffusion statistics, the DiffusionKurtosisFit
+instance can be used to estimate standard non-Gaussian measures as the mean
```

Comment:
=> "...can be used to estimate the non-Gaussian measures of mean kurtosis (MK), axial kurtosis (AK) and radial kurtosis (RK)"